### PR TITLE
Update article with additional Note content for SharePoint mobile app.

### PR DESCRIPTION
**Pull request by jackwi-alt**
Added additional info in Note about exclusively using SVG image for SP mobile app. Also, changed references from 'Office 365' to 'Microsoft 365' per Project Snowbird.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/setup/customize-your-organization-theme.md [hunk 1]
```diff
@@ -59,23 +59,23 @@ Learn how to customize your theme in the Microsoft 365 admin center. As the admi
     
 4. Select **Save changes**.
     
-You'll see your new theme on the admin center right away and after a short delay, you'll see it throughout Office 365, including pages in Outlook, SharePoint, [SharePoint mobile app for iOS](https://support.office.com/en-us/article/SharePoint-mobile-app-for-iOS-339402ce-16bb-4c97-9475-0c5375ccef7a), and [SharePoint mobile app for Android](https://support.office.com/en-us/article/SharePoint-mobile-app-for-Android-d875654b-fb0a-4dbe-a17a-a676cf936284). See the following graphic for an example of where you an customize theme changes from the admin center.
+You'll see your new theme on the admin center right away and after a short delay, you'll see it throughout Microsoft 365, including pages in Outlook, SharePoint, [SharePoint mobile app for iOS](https://support.office.com/en-us/article/SharePoint-mobile-app-for-iOS-339402ce-16bb-4c97-9475-0c5375ccef7a), and [SharePoint mobile app for Android](https://support.office.com/en-us/article/SharePoint-mobile-app-for-Android-d875654b-fb0a-4dbe-a17a-a676cf936284). For an example of where you an customize theme changes from the admin center, see the following image.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/customize-your-organization-theme.md]:
```suggestion
You'll see your new theme on the admin center right away and after a short delay, you'll see it throughout Microsoft 365, including pages in Outlook, SharePoint, [SharePoint mobile app for iOS](https://support.office.com/article/SharePoint-mobile-app-for-iOS-339402ce-16bb-4c97-9475-0c5375ccef7a), and [SharePoint mobile app for Android](https://support.office.com/article/SharePoint-mobile-app-for-Android-d875654b-fb0a-4dbe-a17a-a676cf936284). For an example of where you can customize theme changes from the admin center, see the following image.
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/setup/customize-your-organization-theme.md]: Please change **an** in the last sentence to **can**. Also remove locales from links.

File: microsoft-365/admin/setup/customize-your-organization-theme.md [hunk 2]
```diff
@@ -18,14 +18,14 @@ search.appverid:
...
 ---
 
-# Customize the Office 365 theme for your organization
+# Customize the Microsoft 365 theme for your organization
 
-Learn how to customize your theme in the Microsoft 365 admin center. As the admin of your Office 365 for business subscription, you can change the default theme that appears in the top navigation bar for everyone in the organization. You can add your company logo and change the colors to match the rest of your brand. You can even add a destination link for users to navigate to when they select your logo. You can see here the default theme and custom theme result in Office 365.
+Learn how to customize your theme in the Microsoft 365 admin center. As the admin of your Microsoft 365 for business subscription, you can change the default theme that appears in the top navigation bar for everyone in the organization. You can add your company logo and change the colors to match the rest of your brand. You can even add a destination link for users to navigate to when they select your logo. You can see here the default theme and custom theme result in Microsoft 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/customize-your-organization-theme.md:26]:
```suggestion
Learn how to customize your theme in the Microsoft 365 admin center. As the admin of your Microsoft 365 Apps for business subscription, you can change the default theme that appears in the top navigation bar for everyone in the organization. You can add your company logo and change the colors to match the rest of your brand. You can even add a destination link for users to navigate to when they select your logo. You can see here the default theme and custom theme result in Microsoft 365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/setup/customize-your-organization-theme.md (hunk 2 above)
Comment (jackwi-alt) [reply microsoft-365/admin/setup/customize-your-organization-theme.md:26]: Per Project Snowbird (rebranding Office 365 to Microsoft 365), all instances of 'Office 365 business' change to 'Microsoft 365 Apps for business.'

**Comment by mypil** [comment]
@kwekuako - This is another doc under @twernermsft, would you be the new author for this? 
This PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Comment by kenwith** [comment]
Resolved conflicts and merging.

**Pull request merged**